app/screens/home/search/team_picker_icon/search_team_slideup.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+
+import TeamList from '@components/team_sidebar/add_team/team_list';
+import {useIsTablet} from '@hooks/device';
+import BottomSheetContent from '@screens/bottom_sheet/content';
+import {dismissBottomSheet} from '@screens/navigation';
+
+import type TeamModel from '@typings/database/models/servers/team';
+
+type Props = {
+    teams: TeamModel[];
+    teamId: string;
+    setTeamId: (teamId: string) => void;
+    title: string;
+}
+
+export default function SelectTeamSlideUp({teams, title, setTeamId, teamId}: Props) {
+    const testID = 'team_sidebar.add_team_slide_up';
+    const isTablet = useIsTablet();
+    const showTitle = !isTablet && Boolean(teams.length);
+
+    const onPress = useCallback((newTeamId: string) => {
+        setTeamId(newTeamId);
+        dismissBottomSheet();
+    }, [setTeamId]);
+
+    return (
+        <BottomSheetContent
+            showButton={false}
+            showTitle={showTitle}
+            testID={testID}
+            title={title}
+        >
+            {teams.length &&
```

Comment:
should this component at all return null if the teams length is 0?

Reply:
hmm.. That's an interesting question.  I believe a user must belong to at least one team, so teams.length should never be 0. The question for @matthewbirtch, do we want to disable the team picker if the user only belongs to one team?  I'm going to create this as a Jira ticket for now. https://mattermost.atlassian.net/browse/MM-45623

Reply:
It is possible (although unlikely) that a user is not a member of any teams. However, I don't believe the bottom tab bar (and therefore the search view) is accessible if the user is not a member of a team.